resources/authgear/templates/en/web/authflowv2/__base_page_frame.html

```diff
@@ -27,7 +27,7 @@
   data-controller="prevent-double-tap lockout restore-form loading authflow-passkey-error bot-protection"
   data-restore-form-json-value="{{ $.FormJSON }}"
   data-action="dblclick->prevent-double-tap#action"
-  data-lockout-format-date-relative-outlet="[data-controller='format-date-relative']"
+  data-lockout-format-date-outlet="[data-controller='format-date']"
```

Comment:
We cannot simply change this. If you look at the source code of Lockout Controller, you see it imports the old controller directly, so it has a hard dependency on the old controller.

I suggest we make the lockout controller NOT to depend on other controller. Maybe we can just simply remove the dependency.

Reply:
Tried lockout controller behaves normally without the outlet.

However found another issue where the lockout overlay does not work without js. Will create another issue to follow up